niworkflows/anat/ants.py

```diff
@@ -41,6 +40,8 @@
     "FLAIR": OrderedDict([("nclasses", 3), ("csf", 1), ("gm", 3), ("wm", 2)]),
 }
 
+_ants_version = Registration().version
```

Comment:
This means `import niworkflows.anat.ants` (and possibly `niworkflows`) will try to run a subprocess. I think we should avoid things like that at import. We could keep track of whether we've checked the version with a flag:

```suggestion
_ants_version_checked = False
_ants_version = None
```

Alternately, you could do a one-time property, which returns the cached value after first call:

```Python
from nibabel.onetime import auto_attr
class _version_check:
    @auto_attr
    def ants(self):
        return Registration().version
```

Reply:
Resorted to copy Alejandro's pattern (and implicitly Nipypes memoization of versions) in the workflow for masked datasets a few lines below. Please let me know if that works.